JAndresHJ/react-redux-bootcamp-q2
# Deliverable 3

## What's included?
- Fetch products and creates orders to the provided API
- Search bar functionality to the Products module.
- Add products to the Favorites module.

## How to test it?
Favorites module:
- Click on the heart icon of any product and you should see that the product was added to favorites
- Click on the user icon located at the top right on the Header and click **Favorites**
-  You should see the products of the favorite added with the possibility to remove them.
- Also you can remove a product from favorites by clicking again on the heart icon if the product was already added. This is from the Products page


### Deployed site: https://zingy-pasca-fc36e4.netlify.app/login
